integration/tests/tracer/test_override_params.py

```diff
@@ -0,0 +1,800 @@
+
+    @pytest.fixture(scope="class")
+    def contract_index(self, call_store_value_tx):
+        index = ""
+        address_to = call_store_value_tx["to"].lower()
+        params = self.fill_params_for_storage_contract_trace_call(call_store_value_tx)
+
+        params.append({"tracer": "prestateTracer"})
+        response = self.tracer_api.send_rpc_and_wait_response("debug_traceCall", params)
+
+        assert "error" not in response, "Error in response"
+        for k, v in response["result"][address_to]["storage"].items():
+            if v == padhex(hex(self.storage_value), 64) and k != index_0:
+                index = k
+                break
+        return index
+
+    @pytest.fixture(scope="class")
+    def call_storage_tx(self, storage_contract, web3_client):
+        sender_account = self.accounts[0]
+        _, _, receipt = call_storage(sender_account, storage_contract, self.storage_value, "blockNumber", web3_client)
+        return self.web3_client.wait_get_transaction_by_hash(receipt["transactionHash"].hex())
+
+    @pytest.fixture(scope="class")
+    def call_store_value_tx(self, storage_contract, web3_client):
+        sender_account = self.accounts[0]
+        receipt = store_value(sender_account, self.storage_value, storage_contract, web3_client)
+        return self.web3_client.wait_get_transaction_by_hash(receipt["transactionHash"].hex())
+
+    def retrieve_block_info_tx(self, storage_contract, web3_client):   
+        sender_account = self.accounts[0]
+        nonce = web3_client.eth.get_transaction_count(sender_account.address)
+        instruction_tx = storage_contract.functions.storeBlockInfo().build_transaction(
+            {
+                "nonce": nonce,
+                "gasPrice": web3_client.gas_price(),
+            }
+        )
+        receipt = web3_client.send_transaction(sender_account, instruction_tx)
+        assert receipt["status"] == 1
+
+        return self.web3_client.wait_get_transaction_by_hash(receipt["transactionHash"].hex())
+
+    def fill_params_for_storage_contract_trace_call(self, tx):
+        return [
+            {
+                "to": tx["to"],
+                "from": tx["from"],
+                "gas": hex(tx["gas"]),
+                "gasPrice": hex(tx["gasPrice"]),
+                "value": hex(tx["value"]),
+                "data": tx["input"].hex(),
+            },
+            hex(tx["blockNumber"]),
+        ]
+
+    def test_stateOverrides_debug_traceCall_override_nonce(self, call_storage_tx):
+        params = self.fill_params_for_storage_contract_trace_call(call_storage_tx)
+        params.append({"tracer": "prestateTracer"})
+        response = self.tracer_api.send_rpc_and_wait_response("debug_traceCall", params)
+
+        address_from = call_storage_tx["from"].lower()
+        override_params = {"stateOverrides": {address_from: {"nonce": 17}}, "tracer": "prestateTracer"}
+        params.pop(2)
```

Comment:
We have different tracers, I use `prestateTracer` to check that every overrided param is actually was overrided before emulation. I delete this object `{"tracer": "prestateTracer"}` sometimes cause I want ether to rewrite params or call `tracer` instead of `prestateTracer`